### PR TITLE
Ember 2.11 blog post

### PR DESCRIPTION
**Pull request by locks**
- [Rendered](https://github.com/emberjs/website/blob/release-2-11-0/source/blog/2017-01-23-ember-2-11-released.md).
- [Preview](https://deploy-preview-2784--chess-master-boris-50065.netlify.com/blog/2017/01/09/ember-2-11-released.html).

## Ship approvals

- [x] Ember (@chancancode)
- [ ] Data (@bmac)
- [x] CLI (@nathanhammond)
- [x] Website (@locks, @mixonic)

## Checklist

- Ember
  - [ ] Blog
  - [ ] v2.11.0
  - [ ] v2.12.0-beta.1
- Ember Data
  - [ ] Blog
  - [ ] v2.11.0
  - [ ] v2.12.0-beta.1
- Ember CLI
  - [ ] Blog
  - [ ] v2.11.0
  - [ ] v2.12.0-beta.1

## Instructions (subteam release managers)

* Replace `TBK` with relevant information.
* Check off subteam tasks.

### CODE REVIEW

File: source/blog/2017-01-09-ember-2-11-released.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  adds `factoryFor` as a public API to replace the widely used `_lookupFactory`
+  private API, which is now deprecated. In addition to providing a public API
+  for a sorely needed feature, it also unlocks the opportunity to eliminate one
+  of the major performance hotspot in the Ember.js object-model. This will
+  happen in a future release once the community has had the chance to migrate to
+  the new API. See pull request [#14360](https://github.com/emberjs/ember.js/pull/14360)
+  for additional details.
+
+- [RFC #178](https://github.com/emberjs/rfcs/blob/master/text/0178-deprecate-ember-k.md)
+  deprecates the `Ember.K` utility function. See pull request [#14360](https://github.com/emberjs/ember.js/pull/14360)
```
Comment (Dhaulagiri) [inline source/blog/2017-01-09-ember-2-11-released.md]: i think this should link to https://github.com/emberjs/ember.js/pull/14751

Comment (rwjblue) [inline source/blog/2017-01-09-ember-2-11-released.md]: good spot, thank you!

@locks - Mind updating?

File: source/blog/2017-01-09-ember-2-11-released.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+Node.js LTS.
+
+Following the mitigation section in the recent [security incident
+report](/blog/2016/12/14/security-incident-aws-s3-key-exposure.html) we have
+begun the process of migrating repositories in the [ember-cli GitHub organization](https://github.com/ember-cli)
+to be published automatically via their automated build systems. Ember CLI
+itself has not yet been migrated and as such continues to follow our
+[manual release process](https://github.com/ember-cli/ember-cli/blob/master/RELEASE.md).
+
+#### Ember No Longer Supplied Via `bower`
```
Comment (chancancode) [inline source/blog/2017-01-09-ember-2-11-released.md]: Can you change this to say the positive – i.e. "Ember via npm", or similar. It makes it easier to read imo, but also make this easier to find from the "Please refer the Ember CLI section below for more details." pointer above.

Comment (nathanhammond) [inline source/blog/2017-01-09-ember-2-11-released.md]: I tried but couldn't figure out how to phrase it well with the `npm` and `yarn` split. Wordsmith help please everybody?

Comment (Turbo87) [inline source/blog/2017-01-09-ember-2-11-released.md]: @nathanhammond well, yarn is still using the NPM package registry, so there is no issue with with using the term NPM

File: source/blog/2017-01-09-ember-2-11-released.md [hunk 3]
```diff
@@ -138,17 +138,148 @@ and [addons here](https://github.com/ember-cli/ember-addon-output/compare/v2.10.
...
+now require that addons which need Babel transpilation supply their own
+`ember-cli-babel` dependency. The root application is _no longer_ wholly
+responsible for the transpilation of all dependent addons. There is a
+deprecation message which should guide you through the steps to make the
+necessary changes in your application.
+
+We no longer overload the `babel` key inside of `ember-cli-build.js` to
+conditionally use certain arguments for `babel` and others for
+`ember-cli-babel`. This is categorically better, and we've provided deprecation
+messages to help guide you to the new correct usage pattern.
```
Comment (Turbo87) [inline source/blog/2017-01-09-ember-2-11-released.md]: @nathanhammond isn't that in 2.11 already?

File: source/blog/2017-01-09-ember-2-11-released.md [hunk 4]
```diff
@@ -138,17 +138,148 @@ and [addons here](https://github.com/ember-cli/ember-addon-output/compare/v2.10.
...
+to the newly correct setup.
+
+#### ESLint All The Things!
+
+Tobias Bieniek has been on a mission; we've now completed the move to ESLint
+as the newly recommended linting tool for Ember applications. After upgrading to
+Ember CLI 2.12 and running `ember init` you will be presented with an option to
+remove the existing `ember-cli-jshint` and adopt `ember-cli-eslint`. The process
+should be relatively seamless, please test it out and let us know how it works
+in your applications during the beta period.
```
Comment (Turbo87) [inline source/blog/2017-01-09-ember-2-11-released.md]: - the option that you get is whether it should automatically remove the remaining `.jshintrc` files, but I think `ember-cli-jshint` will be removed unconditionally
- "The process should be relatively seamless" ... unless you made any changes to the defaults and have to port your settings from JSHint to ESLint. Calling it seamless might not be the right term for some people 😉

File: source/blog/2017-01-09-ember-2-11-released.md [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+
+#### Issues with Ember Data Model Fragments
+
+Due to some internal refactorings Ember Data 2.11 has some
+compatibility issues with the popular
+[Ember Data Model Fragments](https://www.npmjs.com/package/ember-data-model-fragments)
+addon. If you are using this addon with your Ember Data application it
+is recommended that you upgrade to
+[Ember Data Model Fragments 2.11](https://github.com/lytics/ember-data-model-fragments/pull/227)
+at the same time you upgrade Ember Data.
```
Comment (bmac) [inline source/blog/2017-01-09-ember-2-11-released.md]: @workmanw @runspired Does this section sound ok to you?

File: source/blog/2017-01-09-ember-2-11-released.md [hunk 6]
```diff
@@ -102,18 +102,83 @@ Ember Data is the offical data persistence library for Ember.js applications.
...
+  serializeId(snapshot, json, primaryKey) {
+    var id = snapshot.id;
+    json[primaryKey] = parseInt(id, 10);
+   }
+});
+```
 
 #### Deprecations in Ember Data 2.12
 
-TBK
```
Comment (chancancode) [inline source/blog/2017-01-09-ember-2-11-released.md]: I think we can ✂️ this section

File: source/blog/2017-01-09-ember-2-11-released.md (hunk 2 above)
Comment (chancancode) [inline source/blog/2017-01-09-ember-2-11-released.md]: :+1: also, the current working makes it sounds like we are dropping support for installing ember.js with `bower`, which is not true.

Perhaps "Installing Ember.js via npm or yarn" if you want to be extra clear

**Comment by rwjblue** [comment]
I have updated the date of the post and tweaked some small verbiage.

The blocking issue (shims related) has been resolved, and updates released in a last round of betas for ember-cli@2.11.0-beta.2 and ember-source@2.11.0-beta.8 (to ensure all is working well together).  Please test those two out and report any issues.

Barring any more unforeseen issues the final release of 2.11.0 is scheduled for 2017-01-23.

File: source/blog/2017-01-23-ember-2-11-released.md [hunk 7]
```diff
@@ -0,0 +1,372 @@
...
+#### Other Notable Changes
+
+- `ember new` and `ember addon` correctly support the `--directory` argument and
+allow you to specify an existing empty directory.
+- Removed `ember-data` and `ember-cli-app-version` from the default
+`ember addon` blueprint.
+- Duplicate calls to `.import('assetname.js)'` will no longer include the asset
+in the output file twice.
+- Given that we no longer require `bower` in the default scenario, we now lazily
+install it into your application the first time that you need it.
```
Comment (Turbo87) [inline source/blog/2017-01-23-ember-2-11-released.md]: @nathanhammond we should add here that global NPM will now be used and requires you to have NPM 3+ installed

Comment (nathanhammond) [inline source/blog/2017-01-23-ember-2-11-released.md]: I'm running a new `git log` and reviewing for adjustments to the language now.

File: source/blog/2017-01-09-ember-2-11-released.md (hunk 3 above)
Comment (nathanhammond) [inline source/blog/2017-01-09-ember-2-11-released.md]: Confirming.

Comment (nathanhammond) [inline source/blog/2017-01-09-ember-2-11-released.md]: Nope, it's a 2.12 feature.

File: source/blog/2017-01-09-ember-2-11-released.md (hunk 4 above)
Comment (nathanhammond) [inline source/blog/2017-01-09-ember-2-11-released.md]: Addressed.

**Comment by mixonic** [comment]
👍

**Pull request merged**